Frends.AzureBlobStorage.ReadBlob/Frends.AzureBlobStorage.ReadBlob/Definitions/Result.cs

```diff
@@ -0,0 +1,12 @@
+﻿#pragma warning disable 1591
+namespace Frends.AzureBlobStorage.ReadBlob
+{
+    public class Result
+    {
+        public string Content { get; set; }
```

Comment:
Please use `private set` instead on public one, since Result-object should be immutable.